frontend/packages/client/src/components/Community/StrategySelectorForm.js

```diff
@@ -0,0 +1,141 @@
+import React, { useEffect, useState } from 'react';
+import { useVotingStrategies } from 'hooks';
+import { AddButton } from 'components';
+import { Bin } from 'components/Svg';
+import { useModalContext } from 'contexts/NotificationModal';
+import StrategyEditorModal from './StrategyEditorModal';
+import { kebabToString } from 'utils';
+
+const StrategyInput = ({
+  index,
+  commuVotStra,
+  onDeleteStrategy,
+  enableDelete,
+  onChange = () => {},
+} = {}) => {
+  return (
+    <div
+      key={`index-${index}`}
+      className="column is-12 is-mobile p-0 m-0 mb-4 fade-in"
+      style={{ position: 'relative' }}
+    >
+      <input
+        type="text"
+        className="border-light rounded-sm p-3 column is-full"
+        value={commuVotStra}
+        onChange={onChange}
+        style={{
+          width: '100%',
+        }}
+      />
```

Comment:
I think we should use a `div` or `p` tag here. It's a weird experience clicking on the row being and seeing a cursor and not able to edit the text.